plugin/unmarshal/unmarshal.go

```diff
@@ -236,6 +236,7 @@ func (p *unmarshal) decodeVarint(varName string, typName string) {
 	p.P(`}`)
 	p.Out()
 	p.P(`}`)
+	p.P(`_ = `, varName) // satisfy ineffassign
```

Comment:
Maybe, but it's generated code. I had this in two locations in maps initially but it left a lot of other places. Not worth optimizing imo.